core/mo-services-impl/ccsds-com-impl/src/main/java/esa/mo/com/impl/archive/db/CallableGetCOMObjects.java

```diff
@@ -43,23 +43,27 @@ public List<COMObjectEntity> call() {
 
         try {
             ResultSet rs;
+            String query = this.transactionsProcessor.dbBackend.getPreparedStatements()
+                                                               .getSelectCOMObjectsQueryString();
             if (this.transactionsProcessor.dbBackend.isPostgres) {
                 // Array-bind is supported in Postgres
-                PreparedStatement stmt = this.transactionsProcessor.dbBackend.getPreparedStatements()
-                        .getSelectCOMObjects();
+                PreparedStatement stmt = c.prepareStatement(query);
```

Comment:
Calling `prepareStatement` on every query goes against prepared statements model and is inefficient -> for Postgres it should be possible to reuse the same prepareStatement as in the original code (but with added `ANY(?)` qualifier)